mathml/elements/mphantom.json

```diff
@@ -36,7 +36,7 @@
               "version_added": "6"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "6"
```

Comment:
I think this should remain `false`, and the subfeature should be `false`, too.
https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mphantom#Browser_compatibility